ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -60,15 +65,38 @@ data = yaml.load(open(path));
 data["compute"] = [ { "name": "worker", "replicas": 0 } ];
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 
+### Update base domain setting
```

Comment:
I think we can pass down the baseDomain value from workflow level.

Reply:
In which step do you mean?

Reply:
I thought gcp also support BASE_DOMAIN env like aws/azure, e.g:
```
- as: installer-rehearse-azuremag
  interval: 960h
  steps:
    cluster_profile: azuremag-qe
    env:
      BASE_DOMAIN: qemag.azure.devcluster.openshift.com
    workflow: cucushift-installer-rehearse-azure-ipi-azuremag
```
After go though ipi-conf-gcp ref, seem like gcp is getting domain from `${CLUSTER_PROFILE_DIR}/public_hosted_zone`, this is different with other platforms.

A new question come out, how did the original dev's xpn workflow to set the baseDomain.

Reply:
As all XPN settings (except credentials), including base domain, had been put in "${CLUSTER_PROFILE_DIR}/qe_gce_xpn.json" (on vault), do you still think we need such a variable? Except XPN, I tend to think different base domain mean different cluster profiles, is it?

Reply:
`upi-conf-gcp-commands.sh` is for both xpn and non-xpn projects, it makes sense that introducing `BASE_DOMAIN`  for it, just like other platforms, this makes all platforms are consistent. Other xpn configurations can be loaded from vault.